src/test/java/org/opensearch/sdk/ExtensionsRunnerForTest.java

```diff
@@ -0,0 +1,34 @@
+package org.opensearch.sdk;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * An Extension Runner for testing using test settings.
+ */
+public class ExtensionsRunnerForTest extends ExtensionsRunner {
+
+    private static final String EXTENSION_DESCRIPTOR = "src/test/resources/extension.yml";
```

Comment:
We'll actually want to remove this. It's further down in the test extension settings.